src/__swaps__/screens/Swap/components/CoinRow.tsx

```diff
@@ -40,15 +37,11 @@ export const CoinRow = ({
   output?: boolean;
   symbol: string;
 }) => {
-  const { AnimatedSwapStyles } = useSwapContext();
   const { favoritesMetadata } = useFavorites();
 
-  const favorites = Object.values(favoritesMetadata);
-
   const isFavorite = (address: string) => {
-    return favorites.find(fav =>
-      fav.address === ETH_ADDRESS ? '0x0000000000000000000000000000000000000000' === address : fav.address === address
-    );
+    // console.log(address, favorites);
+    return Object.values(favoritesMetadata).find(fav => fav.address === address);
```

Comment:
did this to compare addresses, idk if it's useful here if so we can extract to a util
https://github.com/rainbow-me/rainbow/pull/5779/files#diff-642731cd0525a4b6dfdffbd2017e6139bd939d0fad8229022ebb288a7df2b6faR50-R55